pkgs/applications/misc/tthsum/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "tthsum-${version}";
+  version = "1.3.2";
+
+  src = fetchurl {
+    url = "http://tthsum.devs.nu/pkg/tthsum-${version}.tar.bz2";
+    sha256 = "0z6jq8lbg9rasv98kxfs56936dgpgzsg3yc9k52878qfw1l2bp59";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share/man/man1
+    cp share/tthsum.1.gz $out/share/man/man1
+    cp obj-unix/tthsum $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "tthsum is an md5sum-alike program that works with Tiger/THEX hashes";
```

Comment:
Do not repeat the package name in `description`. See http://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.